src/speed-test.js

```diff
@@ -1,6 +1,20 @@
 import { createRandomBuffer } from './utils/random-buffer';
 
+/**
+ * SpeedTest is a class that contains the necessary components
+ * to measure download/upload speed, and ping.
+ */
 export class SpeedTest {
+  /**
+   * @property {string} downloadUrl
```

Comment:
Shouldn't this be @param (constructor's argument)?

Reply:
Absolutely, thanks!

Reply:
Left them as parameters!